### ISSUE TITLE
Package.loadSettings is deprecated.

### ISSUE THREAD

**Issue by florinbroasca** [jroesch/language-scala#24]
Store package settings files in the `settings/` directory instead of `scoped-properties/`
```
Package.loadSettings (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:586:9)
<unknown> (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:188:43)
Package.measure (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:167:15)
Package.load (/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:181:12)
```

**Comment by optikfluffel** [comment]
* duplicate of #14 
* fixed in #12

**Comment by gangstead** [comment]
Fixed in `v1.0.1`
